.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,11 +5,11 @@
 
 ### Fixed Issues
 <!---
-1. Please replace GH_LINK with a URL link to the GitHub issue this Pull Request is fixing.
-2. Please replace PROPOSAL: GH_LINK_ISSUE(COMMENT) with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify).
+1. Please postfix `$` with a URL link to the GitHub issue this Pull Request is fixing.
+2. Please postfix  `PROPOSAL:` with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify).
```

Comment:
```suggestion
1. Please postfix `$` with a URL link to the GitHub issue this Pull Request is fixing. For example, `$ GH_LINK`.
2. Please postfix  `PROPOSAL:` with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify). For example, `PROPOSAL: GH_LINK_ISSUE(COMMENT)`
```

should we keep the example formats?

Reply:
That would kind of be 2 steps forward and 1 step backward 😅 What about: 
```suggestion
1. Please postfix `$` with a URL link to the GitHub issue this Pull Request is fixing. For example, `$ https://github.com/Expensify/App/issues/123`.
2. Please postfix  `PROPOSAL:` with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify).  For example, `PROPOSAL: https://github.com/Expensify/App/issues/123#issuecomment-1369752925`
```

Reply:
Something like that makes sense! I think it's clear enough without the example, but I'm trying to think from the perspective of first-time contributors

Reply:
Updated